### PR TITLE
OpenBSD: Added support for static routes

### PR DESCRIPTION
**Pull request by kadiron**
## Proposed Commit Message
OpenBSD: Added support for static routes

```
This adds support for static routes on OpenBSD.
Note that since BSDs see routes as per-interface-configurations, you have to specify your routes inside the network interface context.
```

### CODE REVIEW

File: cloudinit/net/openbsd.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def start_services(self, run=False):
                     ["dhcpleasectl", "-w", "30", interface], capture=True
                 )
         else:
-            net.dhcp.IscDhclient.kill_dhcp_client()
+            subp.subp(["pkill", "dhclient"], capture=True, rcs=[0, 1])
```
Comment (goneri) [inline cloudinit/net/openbsd.py:46]: Hi, I'm not sure this needed. `kill_dhcp_client()` does the same thing already.

File: cloudinit/net/openbsd.py [hunk 2]
```diff
@@ -54,6 +54,12 @@ def set_route(self, network, netmask, gateway):
             fn = subp.target_path(self.target, if_file)
             content = gateway + "\n"
             util.write_file(fn, content)
+        else:
+            try:
```
Comment (goneri) [inline cloudinit/net/openbsd.py:58]: It would be more elegant to define the attribute in the constructor.

**Review by goneri — CHANGES_REQUESTED** [review]

File: cloudinit/net/openbsd.py (hunk 1 above)
Comment (kadiron) [reply cloudinit/net/openbsd.py:46]: Hi and sorry, you're right. I don't know what's happened here, I didn't wanted to change this. It's the line as it is in my clone from main at that day, so just ignore that please.

File: cloudinit/net/openbsd.py (hunk 2 above)
Comment (kadiron) [reply cloudinit/net/openbsd.py:58]: Sure, I'm aware of this being hacky. But tbh, I don't know how to do this right -- I hardly ever had to do anything in python yet.

**Comment by kadiron** [comment]
Here is one version conflict and something hacky that shouldn't be like this. I'm starting over and will make a new pull request.
Thanks so far!

**Pull request closed without merging**